hazelcast-client/src/main/java/com/hazelcast/client/proxy/NearCachedClientMapProxy.java

```diff
@@ -52,8 +52,6 @@
 import static com.hazelcast.map.impl.MapListenerFlagOperator.ALL_LISTENER_FLAGS;
 import static java.util.Collections.emptyMap;
 
-import java.util.ArrayList;
```

Comment:
Can someone have a look at this please? I have no idea what is going on, but the CI build is failing on a checkstyle problem which does not actually exist:
`12:49:57 /scratch/jenkins/workspace/Hazelcast-3.x-pr-builder/hazelcast-client/src/main/java/com/hazelcast/client/proxy/NearCachedClientMapProxy.java:42:8: Unused import - java.util.ArrayList`

This PR is otherwise good to go, and has already been reviewed.